components/notability/commons/notability_checker.lua

```diff
@@ -155,6 +154,25 @@ function NotabilityChecker._calculateWeight(placementData)
 	return finalWeight
 end
 
+function NotabilityChecker.calculateTournament(liquipediatier, liquipediatiertype, placement, date, notabilityMod, mode)
+	local dateLoss = NotabilityChecker._calculateDateLoss(date)
+	notabilityMod = NotabilityChecker._parseNotabilityMod(notabilityMod)
+	local tier, tierType = NotabilityChecker._parseTier(liquipediatier, liquipediatiertype)
```

Comment:
Variables need a better name

Reply:
Which specific variables do you think need a new name, and got any suggestion for their new name?

(nb: variable names seen in the conversation are outdated, as they have been slightly changed)

Reply:
The two that are commented on, `tier` and `tierType`. Don't give new values to input vars

Reply:
Updated